pkgs/by-name/op/opensta/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  tcl,
+  swig,
+  bison,
+  flex,
+  git,
+  cudd,
+  eigen,
+  libzip ? null,
```

Comment:
Could you explain why this needs a `null` default value?

Reply:
It doesn't. I thought it was necessary for having libzip as an optional dependency. But actually there is no need to have it as an optional, it should be available on all platforms.